### PR TITLE
Feature/google analytics

### PR DESCRIPTION
**Pull request by borysyuk**

### CODE REVIEW

File: config/nunjuck.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const path = require('path');
+require('dotenv').config();
```
Comment (vitorbaptista) [inline config/nunjuck.js]: Add the `require('dotenv').config()` before the `const path = require('path')`. This won't change much here, but it's good to keep the pattern of, when needed, running `dotenv` before anything else.

File: .env.example [hunk 2]
```diff
@@ -15,3 +15,4 @@ S3_BUCKET=s3-bucket
 S3_REGION=s3-region
 S3_CUSTOM_DOMAIN=https://s3-custom-domain-if-it-exists.com
 MAX_UPLOAD_SIZE=size-in-bytes
+GOOGLE_ANALYTICS_CODE=
```
Comment (vitorbaptista) [inline .env.example]: Just to keep it consistent with the rest of the file, could you change it to something like `GOOGLE_ANALYTICS_CODE=google-analytics-code`

**Comment by vitorbaptista** [comment]
All good to go, just these minor adjustments :+1:

**Comment by borysyuk** [comment]
@vitorbaptista done.

**Comment by vitorbaptista** [comment]
Merged :+1:

**Pull request merged**